File: cpp/workspaces/warehouse/src/core/Item.cpp
#include "Item.h"

namespace core {
    Item::Item(int id, const std::string& name, int quantity)
        : id_(id), name_(name), quantity_(quantity) {}

    int Item::getId() const {
        return id_;
    }

    const std::string& Item::getName() const {
        return name_;
    }

    int Item::getQuantity() const {
        return quantity_;
    }

    void Item::setQuantity(int quantity) {
        quantity_ = quantity;
    }
} // namespace core

File: Windows/dotnet4-x/workspaces/warehouse/src/Core/Item.cs
using System;

namespace App
{
    public class Item
    {
        public int Id;
        public string Name;
        public int Quantity;

        public Item(int id, string name, int quantity)
        {
            Id = id;
            Name = name;
            Quantity = quantity;
        }
    }
}